.zenodo.json

```diff
@@ -0,0 +1,19 @@
+{
+    "creators": [
+        {
+            "affiliation": "Netherlands eScience Center",
+            "name": "Rol, Evert",
+            "orcid": "0000-0001-8357-4453"
+        },
+        {
+            "affiliation": "Netherlands eScience Center",
+            "name": "Kalverla, Peter",
+            "orcid": "0000-0002-5025-7862"
```

Comment:
At some point perhaps also add Jaro. :-)

Reply:
Yes, I thought I'd add him for the big update that we've been preparing. Agree @JaroCamphuijsen ?